catalog/files/impl/src/main/java/org/projectnessie/catalog/files/s3/S3Utils.java

```diff
@@ -32,7 +32,7 @@ public final class S3Utils {
   private S3Utils() {}
 
   public static Optional<String> extractBucketName(URI uri) {
-    String scheme = requireNonNull(uri).getScheme();
+    String scheme = requireNonNull(uri, "URI argument missing").getScheme();
```

Comment:
Couldn't we settle on a standard message format? E.g.. `argument %s is null`

Reply:
At least we have something that helps more than just an NPE w/o a reason